packages/yew-macro/src/derive_props/builder.rs

```diff
@@ -120,7 +120,10 @@ impl PropsBuilder<'_> {
             .filter(|pf| pf.is_required())
             .map(|pf| pf.to_step_name(prefix))
             .collect();
-        step_names.push(Ident::new(&format!("{}_build", prefix), Span::call_site()));
+        step_names.push(Ident::new(
+            &format!("{}PropsBuilder", prefix),
+            Span::call_site(),
```

Comment:
Nit: change this to `prefix.span()` maybe? It doesn't really matter though

Reply:
I tried that, but I get an error: `proc_macro2::Span` cannot be formatted with the default formatter.

If it's just a nit, I'll leave it at that if you don't mind :)

Reply:
I think you put that in the format. You have to replace `Span::call_site()` with it

Reply:
Ah, got it! Fixed now :)